tests/e2e/storage.e2e-spec.ts

```diff
@@ -0,0 +1,420 @@
+require('dotenv').config();
+import request from 'supertest';
+import { HttpStatus } from '@nestjs/common';
+import { encryptFilename, createTestUser, deleteTestUser } from './utils';
+import { Sign } from '../../src/app/middleware/passport';
+import { applicationInitialization } from './setup';
+import { FileModel } from '../../src/app/models/file';
+import { FolderModel } from '../../src/app/models/folder';
+import sequelize from 'sequelize';
+const { Op } = sequelize;
+const server = require('../../src/app');
+const app = server.express;
+
+const deleteAllUserFilesFromDatabase = async (userId: number): Promise<void> => {
+  const files = await server.models.file.findAll({ where: { user_id: userId } });
+  await files.forEach((file: FileModel) => file.destroy());
+};
+
+const deleteAllUserFoldersFromDatabase = async (userId: number): Promise<void> => {
+  const folders = await server.models.folder.findAll({ where: { user_id: userId } });
+  await folders.forEach((folder: FolderModel) => folder.destroy());
+};
+
+const clearUserDrive = async (userId: number, rootFolderId: number): Promise<void> => {
+  await deleteAllUserFilesFromDatabase(userId);
+
+  const folders = await server.models.folder.findAll({ where: { user_id: userId, id: { [Op.not]: rootFolderId } } });
+  await folders.map((folder: FileModel) => folder.destroy());
+
+  await server.database.query('DELETE FROM deleted_files WHERE user_id = (:userId)', {
+    replacements: { userId },
+  });
+};
+
+const createFolder = async (body: any, authToken: string): Promise<request.Response> =>
+  await request(app).post('/api/storage/folder').set('Authorization', `Bearer ${authToken}`).send(body);
+
+const createFileOnFolder = async (folderId: number, name: string, authToken: string): Promise<request.Response> =>
```

Comment:
I also like to do these re-usable functions for tests, makes it much easier to read